cloudfoundry/stratos
# TEAMFOUR-40: Register clusters and services

- added service registration styling for cluster and service registration (using mock data)
- added serviceStatus filter that converts a string status into HTML markup for status icon
- added unit tests and documentation

To test the UI:
1. Login, you should see the registration page.
2. Click on "Enter Credentials" for a cluster. That will enable that cluster and enable the "Done" button.
3. Click "Done" and it should redirect you to the default "Applications" page.
